SCClassLibrary/Common/Control/Server.sc

```diff
@@ -670,36 +578,53 @@ Server {
 		^Bus(\audio, 0, this.options.numOutputBusChannels, this);
 	}
 
-	startAliveThread { arg delay=0.0;
-		this.addStatusWatcher;
-		^aliveThread ?? {
-			aliveThread = Routine({
-				// this thread polls the server to see if it is alive
-				delay.wait;
-				loop({
-					this.status;
-					aliveThreadPeriod.wait;
-					this.serverRunning = alive;
-					alive = false;
-				});
-			});
-			AppClock.play(aliveThread);
-			aliveThread
-		}
+
+	/* backward compatibility */
```

Comment:
I wouldn't refer to this as "backward compatibility". the statusWatcher should be considered a private object and people should not be encouraged to use it or to think that its the new way to do things.

these methods are the correct way IMO to expose the api. so I suggest removing the "backward" comment